WorkloadManagementSystem/Service/JobManagerHandler.py

```diff
@@ -118,6 +119,8 @@ def export_submitJob( self, jobDesc ):
     parametricJob = False
     if nParameters > 0:
       parametricJob = True
+      if nParameters > self.maxParametricJobs:
```

Comment:
Isn't nParameters here the number of Parameters, not the number of Jobs that come out in the end?

The check has to go into the generateParametricJobs function

Reply:
The number of jobs to generate is always equal to the number of parameters. So, there is no point to attempt to generate jobs if the number of parameters exceeds the limit. I do not think that you want to generate 100 first jobs with the limit = 100 while your JDL contains 200 parameters.

Reply:
Are you calling number of parameters what is actually number of parameter values?